samples/traceapi/_win32.cpp

```diff
@@ -4319,8 +4319,10 @@ BOOL (__stdcall * Real_GetThreadContext)(HANDLE a0,
 HDESK (__stdcall * Real_GetThreadDesktop)(DWORD a0)
     = GetThreadDesktop;
 
+#if(WINVER >= 0x0500)
```

Comment:
No need for parens.

Reply:
Also, can declare the function yourself.
Assuming the import libs have no such filtering.
I'm not sure which is best. Versioning is an unsolved mess (except for just staying at head like some large orgs..)